test/libsolidity/syntaxTests/viewPureChecker/inline_assembly_instructions_disallowed_pure.sol

```diff
@@ -32,7 +32,7 @@ contract C {
             pop(coinbase())
             pop(timestamp())
             pop(number())
-            pop(difficulty())
+            pop(prevrandao())
```

Comment:
Not sure if I just missed it, but looks to me like a ``<paris`` test confirming that ``difficulty()`` is disallowed in pure functions is missing, isn't it?

Reply:
Indeed. I added it.